javascript/react-js/inputs-and-lists.md

```diff
@@ -0,0 +1,343 @@
+### Introduction
+
+Now before we actually go any further, let's do a quick check of how well you understand the topic. Following I will describe the quick assignment, for those who want to try it themselves. I encourage everyone to at least try it yourself, if you've done the reading from the last lectures, you should be able to do it. And if not, you can always come back to this section, because we will discuss the assignment in detail together. Don't be disscouraged if you can't solve it, it includes some tricky parts that we haven't actually discussed so far.
+
+Before you start, make sure you understood the concept of `state` and `props` from the previous lessons.
+
+### Learning Outcomes
+
+- Understanding how to render lists in React
+- Understanding how to handle input fields and forms in React.
+
+### Assignment
+
+### DIY Guide
+
+Our application will be made of two components, `Tasks` and `Display`. Your application should render an input field and a submit button. With the submit button you can add the content from your input to a task array managed in state. (So we will use class components, because we haven't learned about hooks yet). Your application should render a html list element for each element in the task array and display it on the screen.
```

Comment:
per @heldersrvio - The `Tasks` and `Display` are never mentioned again here, or in the detailed guide.
Making note for a rewrite or removal of redundant information.

Reply:
Yes seems like I forgot those when renaming the components to Overview and App. Thanks for mentioning.